process-migration-service/pom.xml

```diff
@@ -631,6 +633,8 @@
                     <argument>-jar</argument>
                     <argument>${project.build.directory}/${project.build.finalName}-thorntail.jar</argument>
                     <argument>-s${project.basedir}/src/test/resources/project-test.yml</argument>
+                    <argument>-Dthorntail.security.security-domains.pim.classic-authentication.login-modules.UsersRoles.module-options.usersProperties=${project.basedir}/src/test/resources/application-users.properties</argument>
+                    <argument>-Dthorntail.security.security-domains.pim.classic-authentication.login-modules.UsersRoles.module-options.rolesProperties=${project.basedir}/src/test/resources/application-roles.properties</argument>
```

Comment:
What about just putting the values into the `project-test.yml` file? Then we don't have do duplicate this -> better maintainability. Plus even in the Thorntail docs they say that system properties-based approach is rather a short-term solution.

Reply:
what would be the path I should use? I couldn't make it work

Reply:
It's the same as here, but you have to move the project-test.yml to the filtered-resources folder and add this to the pom.xml:

```
.
.
.
<build>
    <testResources>
      <testResource>
        <directory>src/test/filtered-resources</directory>
        <filtering>true</filtering>
      </testResource>
    </testResources>
.
.
.
```
Then the Maven will take care of the rest.